RHEL6_7/networking/net-snmp/check

```diff
@@ -12,7 +12,7 @@ In most configurations, no changes to the configuration files are necessary.
 See the following knowledge base article at [link:https://access.redhat.com/site/articles/696163] for any known incompatibilities.
 Retest carefully all applications consuming SNMP data from this system with the updated Net-SNMP package.
 _EOF_
-    exit $RESULT_INFORMATIONAL
+    exit $RESULT_FAIL
 fi
 
```

Comment:
It's ok. But what do you think about use of `log_slight_risk` above? I am not familiar with incompatibilities, so not sure whether it should be slight or medium risk. You can keep it as it is, but maybe we should create somewhere note about that to investigate it later.

Reply:
Based on the KB and snmpd.conf man page, I assume it can cause the issue in certain corner cases. It can potentially affect some associated sysadmin scripts. I will change the risk to `log_medium_risk`